test/lib/pgBackRestTest/Full/FullSyntheticTest.pm

```diff
@@ -831,22 +837,6 @@ sub run
             $strBackup, \%oManifest, undef, $bDelta, $bForce, undef, undef, undef, undef, undef, undef,
             'fail on used path', ERROR_PATH_NOT_EMPTY, '--log-level-console=detail');
 
-        # Fail on undef format
```

Comment:
I guess you decided these tests were unnecessary? (fail on undef format; fail on mismatch format)